src/main/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/repository/UserRepository.kt

```diff
@@ -24,4 +24,7 @@ interface UserRepository : CoroutineCrudRepository<User, UUID> {
   fun findAllByGroupCode(groupCode: String, source: AuthSource = AuthSource.auth): Flow<User>
 
   fun findByEmailAndSourceOrderByUsername(email: String?, source: AuthSource = AuthSource.auth): Flow<User>
+
+  @Query("SELECT * FROM USERS WHERE source = :source")
```

Comment:
Is this SQL query needed? Does the default findAllBySource do this automatically?